### DISCUSSION TITLE
Combining entering animations

### DISCUSSION THREAD

**Discussion by wallacerenan** [software-mansion/react-native-reanimated#5668] [Q&A]
Any way to combine entering animations?

Like:

```typescript
     <Animate.View
         entering={FadeIn.delay(200).duration(800).Fade.duration(800)}
     />
```

**Comment by MatiPl01** [comment]
You cannot combine predefined layout animations in this way. You can either apply both layout animations to separate components (nest `View` components with different `entering` animation) or better, write the custom layout animation yourself (see [docs](https://docs.swmansion.com/react-native-reanimated/docs/layout-animations/custom-animations) for more details).
